src/Wallabag/ImportBundle/Import/BrowserImport.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+namespace Wallabag\ImportBundle\Import;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Doctrine\ORM\EntityManager;
+use Wallabag\CoreBundle\Entity\Entry;
+use Wallabag\UserBundle\Entity\User;
+use Wallabag\CoreBundle\Helper\ContentProxy;
+
+class BrowserImport implements ImportInterface
```

Comment:
You should extends `AbstractImport` instead.
I moved some stuff to it to avoid duplicate method / variable between imports (like constructor, setUser, setLogger, etc).